### ISSUE TITLE
x/pkgsite:  several img tags lack descriptive alt text

### ISSUE THREAD

**Issue by jba** [golang/go#41104]
Current alt text says "Go" which is fine, but adds no semantic content for screen reader users. "Go logo" would be preferable.

**Comment by jba** [comment]
First comment applies to `<img class=”NavigationDrawer-logo”>` in header.

Also:
-  `<img class=”Header-logo”>` in header: use "Go logo"
- `<img class=”Footer-gopher”>` in footer: use "Gopher in flight goggles"

**Comment by jimmyfrasche** [comment]
.Header-logo and .NavigationDrawer-logo are the sole contents of links so the alt text should describe the destination of the link.

.Footer-gopher is purely decorative so its alt text should (explicitly) be the empty string.
